src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg" 
+        alt="Apple A1419 iMac 27 Retina 5K Monoblock"
+        class="product-card__img"
+      >
+
+      <p class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="product-card__product-code">
+        Product code: 195434
+      </p>
+
+      <div class="rating">
+        <div class="rating__stars">
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+          <div class="rating__star"></div>
+        </div>
```

Comment:
```suggestion
      <div class="card__rating">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
```

Reply:
you should have 2 BEM blocks card and stars
and keep styles for them in two separate files as well